guides/process/support/triage.md

```diff
@@ -0,0 +1,78 @@
+# Support Triage
+
+Triage is the usual entry point to the support process. Customers will email their request to support@madetech.com, which is routed to [Zendesk](https://madetechsupport.zendesk.com).
+
+On ocassion customers will ask for things via Slack or phone, in which case we can create the ticket on their behalf in Zendesk.
+
+## Process Overview
+
+It is the responsibility of the Triage Pair to:
+
+- [ ] 1. Triage the oldest item from the queue
+- [ ] 2. Record time in Harvest
+- [ ] 3. Ensure we're meeting our Service Level Agreements
```

Comment:
No formal SLA. I was thinking the acknowledgement in 1 day and resolution in 2 as listed below was enough to start?